framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Grid/Grid.module.scss

```diff
@@ -41,6 +41,11 @@ $flex-align-items: 'flex-start', 'center', 'flex-end', 'stretch', 'baseline';
 	width: 100%;
 }
 
+.gridRowNoWrap {
+	display: flex;
+	width: 100%;
+}
+
```

Comment:
I am ok with having this additional option. But if you keep the number of columns in the row consistent to 12 they will never wrap anyway. If you think that's fine then we can remove that additional option.